test/test_api.py

```diff
@@ -64,6 +64,7 @@ class APICheck2 {
         m2 = API.Instance_FromVoidPtr(voidp, 'APICheck2')
         assert m is m2
 
+    @mark.xfail(run=not (IS_LINUX_ARM), reason="Crashes pytest on Linux ARM")
```

Comment:
```suggestion
    @mark.xfail(run=False, condition=IS_LINUX_ARM, reason="Crashes pytest on Linux ARM")
```

You will have to make similar changes in all the locations that add this xfail.

When a test fails in a specific platform we use `condition` to specify which platform it is failing on. And set `run` depending on whether it crashes or not. If it crashes `run=False`.
If a test fails in all platforms but only crashes in Linux ARM, then `run=not IS_LINUX_ARM`, and need not specify `condition`.

Reply:
Thanks for the suggestion. Will make this later today, or possibly tomorrow